README.md

```diff
@@ -1 +1,103 @@
-# deployment-deploy-services
\ No newline at end of file
+# How to deploy dalite-ng on OpenStack server
+ 
+This repository contains ansible deployment scripts allowing one to deploy dalite-ng to OpenStack server.
+
+
+Create a OpenStack environment
+------------------------------
+
+1. Create a root volume that has 40GB size. It should be based on Ubuntu 14.04 image. 
+   We suggest you use vanilla image from [ubuntu page](https://cloud-images.ubuntu.com/). 
+   To do that go to (`Volumes` -> `Create Volume` -> `Volume Source` choose `Image` -> `Select image`). 
+2. Create `vps-ssd-3` instance from this volume: (`Boot Source` -> `Volume` -> `Your root volume`). 
+3. Go to info page and note the instance public key, ssh to the instance and check whether public key matches, 
+   save public key to your ssh config.
+4. Create a `hosts` file containing: 
+
+        [dalite]
+        dalite.harvardx.harvard.edu ansible_host=149.202.190.106
+                
+   Please note that while host name is largely irrelevant, this host must be in dalite group.   
+
+Obtain database credentials
+---------------------------
+
+1. Create a `private-extra-vars.yml` file and store it somewhere safe, all configuration should be stored in that file. 
+2. Obtain database credentials 
+3. Save host to `MYSQL_DALITE_HOST`
+4. Save database to: `MYSQL_DALITE_DATABASE`
+5. Save credentials to: `MYSQL_DALITE_PASSWORD`, `MYSQL_DALITE_USER`
+
+   
+Generate secrets
+----------------
+
+1. Generate tarsnap keys, these keys shouldn't end up on dalite-ng server, but instead store them somewhere safe. 
+    1. Generate key for swift container backup 
+    2. Generate key for dalite logrotate backup   
+2. Generate tarsnap read write keys from the master key, see [tarsnap-keymngmt](http://www.tarsnap.com/man-tarsnap-keymgmt.1.html),    
+3. Go to dead man's snitch at https://deadmanssnitch.com/. And generate three snitches, save them under:
```

Comment:
I'll amend the readme, to unblock you: 

1. DALITE_LOG_TARSNAP_SNITCH - daily
2. SANITY_CHECK_SNITCH 15min
3. BACKUP_SWIFT_SNITCH hourly